include/moveit_grasps/grasp_generator.h

```diff
@@ -97,13 +97,13 @@ enum grasp_axis_t
 struct GraspCandidateConfig
 {
   GraspCandidateConfig()
-    : enable_corner_grasps_(false)
-    , enable_face_grasps_(false)
-    , enable_variable_angle_grasps_(false)
-    , enable_edge_grasps_(false)
-    , generate_x_axis_grasps_(false)
-    , generate_y_axis_grasps_(false)
-    , generate_z_axis_grasps_(false)
+    : enable_corner_grasps_(true)
+    , enable_face_grasps_(true)
+    , enable_variable_angle_grasps_(true)
+    , enable_edge_grasps_(true)
+    , generate_x_axis_grasps_(true)
+    , generate_y_axis_grasps_(true)
+    , generate_z_axis_grasps_(true)
```

Comment:
why are you default all these to true?

Reply:
It preserves the same behavior as before and is closer to what a user might expect out of the box. I was confused for a second why the grasp generator generated 0 grasps for a demo and I made these changes. I can only imagine that a potential user would be much more confused.

Reply:
this is very subjective and not what i believed when i wrote this code, but it doesn't matter enough and this is fine to me now